### DISCUSSION TITLE
Is there a way to load a plugin after alpine is loaded.

### DISCUSSION THREAD

**Discussion by cameronbraid** [alpinejs/alpine#3998] [4. General]
I wand to do lazy loading of plugins on demand using import().then(..)

**Comment by calebporzio** [comment]
You can lazyilly load a plugin right now, but it won't re-initialize the page and look for instances it could have been used - it will only use that plugin for elements that have been refreshed or initialized after you load the plugin

**Comment by ekwoka** [comment]
You can load them whenever you want

Just needs to be before Alpine initializes those elements (or you manually destroy and reinit the tree)
